### DISCUSSION TITLE
How would eslint be set up?

### DISCUSSION THREAD

**Discussion by hello-smile6** [hello-smile6/freecat#2] [Q&A]
@webdev03 What would we do?

**Comment by imdevarsh** [comment]
I'm not very sure, but https://eslint.org/docs/user-guide/getting-started and also there is a plugin for Prettier support, so:
Prettier formats and lints, ESLint checks if you have formatted and if not gives errors.

**Reply by hello-smile6** [reply]
Could you help set it up?

**Reply by imdevarsh** [reply]
Sure, I can PR into here.

**Reply by imdevarsh** [reply]
Why can't I fork a fork? I don't even have a VPN, or anything

**Reply by hello-smile6** [reply]
> Why can't I fork a fork? I don't even have a VPN, or anything

I don't either. I just deal with it and give people my IP.

**Reply by hello-smile6** [reply]
> Sure, I can PR into here.

Could you add eslint?

**Reply by imdevarsh** [reply]
I don't want to contribute to the origin freecat right now, I might do it here.

**Reply by hello-smile6** [reply]
Could you?

**Reply by imdevarsh** [reply]
OK!

**Reply by hello-smile6** [reply]
@webdev03 Want me to just wipe all commits and push a blank repository containing nothing but the current version?

**Reply by imdevarsh** [reply]
No
